.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/Users/zachcm/miniconda3/envs/pyme/bin/python"
```

Comment:
could you remove this file please. Can add .vscode/* to .gitignore if you like

Reply:
Yep. Sorry about that.